### DISCUSSION TITLE
New name for TuxGuitar?

### DISCUSSION THREAD

**Discussion by guiv42** [helge17/tuxguitar#286] [General]
Hi everyone.

Opening this discussion because a modification of application name is required for this fork to be integrated:
- in F-Droid  for Android version, see #138 
- in brew for macOS version, see [this thread](https://github.com/Homebrew/homebrew-cask/pull/169109#issuecomment-1999186081)

In both cases conditions for new forks are explicit and clear, so we need to comply if we want to go forward. Not a simple decision though.

I see the following open points:
1. what new name?
2. when?
3. shall we also update name of Windows & Linux versions?

My personal opinion:
1. name: I suggest **TuxGuitar+**. It's short, simple, and it shows continuity with the original project
2. when: not urgent I think. Could be done together with the first version breaking ".tg" file format? (needed for new features). That should delay introduction in brew, but that would be consistent.
3. Linux & Windows: It seems logical to change name of all variants, for consistency. But **TuxGuitar** 1.6.1 is already deployed in openSuse, renaming package may not be so simple. Should we wait this new name before submitting this fork to other distros? I don't know.

**Comment by belfie13** [comment]
I don't know but my 2 cents is to use TuxGuitar+ where they require a new name and leave it be elsewhere.

one thing, on those platforms where we add the `+`, can you find it with a search for `tuxguitar`. I don't have access to those platforms so might be good to check by finding another app with `+` appended to the name and searching for it without the `+`. example: app name = `someapp+` search = `someapp`, and see if it comes up. then no problems. otherwise if you can't search for "tuxguitar" on all platforms and find us then we should change the name (which i don't like at all, we should keep  "tuxguitar" if possible)
